components/frontend/src/header_footer/CollapseButton.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Button, Icon } from 'semantic-ui-react';
+import { Popup } from '../semantic_ui_react_wrappers';
+import { stringsURLSearchQueryPropType } from '../sharedPropTypes';
+
+export function CollapseButton({ expandedItems }) {
+    const label = "Collapse all headers and metrics"
+    return (
+        <Popup
+            on={["hover", "focus"]}
+            trigger={
+                <span  // We need a span here to prevent the popup from becoming disabled whenever the button is disabled
```

Comment:
Still needed? Is the empty line needed for comment or can it be `<span>`?

Reply:
`<span>  // Comment` doesn't work unfortunately.